docs/guide/core/object.md

```diff
@@ -1,67 +1,69 @@
 
-初始化一个`Object3D`对象之后，默认处于启用状态，通过设置`enable`变更节点的状态，如果设置为`false`，这将停用该节点对象以及该节点下所有的子节点对象，同时附加在这些节点之上的所有组件也将停止调用。
 ```ts
 let obj = new Object3D();
 obj.enable = false;
 ```
 
-## 节点的添加和移除
-添加一个子节点，可以使用 [addChild](/api/classes/Object3D#addchild) 方法   
-移除一个子节点，可以使用 [removeChild](/api/classes/Object3D#removeChild) 方法   
-移除特定位置子节点，可以使用 [removeChildByIndex](/api/classes/Object3D#removeChildByIndex) 方法   
-从父级移除自己，可以使用 [removeFromParent](/api/classes/Object3D#removeFromParent) 方法   
-移除所有子节点，可以使用 [removeAllChild](/api/classes/Object3D#removeAllChild) 方法   
+## Add and Remove a Node
+
+To add a child node, you can use [addChild](/api/classes/Object3D#addchild) method
+To delete a child node, you can use [removeChild](/api/classes/Object3D#removeChild) method
+To remove a child node at a specific position, you can use [removeChildByIndex](/api/classes/Object3D#removeChildByIndex) method
+To remove this from the parent, you can use [removeFromParent](/api/classes/Object3D#removeFromParent) method
+To remove all child nodes, you can use [removeAllChild](/api/classes/Object3D#removeAllChild) method
 
 ```ts
 let parent = new Object3D();
 let child = new Object3D();
-//添加节点
+//Add a child node
 parent.addChild(child);
-//移除节点
+//Remove the node
 parent.removeChild(child);
-//or 子节点自己删除
+//or remove the child node by itself
 child.removeFromParent();
-//or 移除所有子节点
+//or remove all child nodes
 parent.removeAllChild();
 ```
 
-## 添加和移除组件
-通过 `Object3D` 自带的 [addComponent](/api/classes/Object3D#addComponent) 和 [removeComponent](/api/classes/Object3D#removeComponent) 方法，可以方便的在运行时添加和移除组件。
+## Add and Remove Components
+Using the built-in [addComponent](/api/classes/Object3D#addComponent) and [removeComponent](/api/classes/Object3D#removeComponent) methods of `Object3D`, you can easily add and remove components when calling them.
 
 ```ts
 let obj = new Object3D();
-//为节点添加平行光组件
+//Add a parallel light component to the node
 let dl = obj.addComponent(DirectLight);
-//移除组件
+//Remove the component
 obj.removeComponent(DirectLight);
 ```
 
-## 获取组件节点
-所有的组件都是继承于 `ComponentBase`，在组件内部可以通过 `this.object3D` 属性获得当前组件所属的节点。
+## Get Component Node
+All the components are extended from `ComponentBase`, and you can get the current component through the `this.object3D` property inside the component.
 ```ts
-//自定义一个组件，效果是把节点的X轴位置增加10
+//Custom component, to increase the X-axis position of the node by 10
 class CustomComponent extends ComponentBase {
     public start() {
         this.object3D.x += 10;
     } 
 }
 ```
 
-## 获取节点的其它组件
-通过 `Object3D` 自带的 [getComponent](/api/classes/Object3D#getComponent) 方法，可以方便的获取节点上的组件。
+## Get Other Components of Node
+Using the built-in [getComponent](/api/classes/Object3D#getComponent) method of `Object3D`, you can easily get the components on the node.
 ```ts
-//自定义一个组件，效果是变更节点上的另一个灯光组件，改变灯光的颜色
+//Custom component, to change another light component on the node, change the color of the light
```

Comment:
Customize a component.

Double-check the purpose of this coding block: is it to change another light component's color on the node

Reply:
I'm not sure, but the chinese document says so